### PR TITLE
JSUI-2402 Add Unit Tests for MLFacet Component

### PR DESCRIPTION
**Pull request by ThibodeauJF**
https://coveord.atlassian.net/browse/JSUI-2402

[Deploy](https://dashboard.heroku.com/pipelines/a3535101-5bbf-4a5b-a909-47fcf8c9f149)

### CODE REVIEW

File: unitTests/ui/MLFacet/MLFacetTest.ts [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+
+    it('title option should set the title', () => {
+      options.title = 'a title';
+      initializeComponent();
+      test.cmp.ensureDom();
+
+      expect($$(test.cmp.element).find('.coveo-facet-header-title').innerHTML).toBe(options.title);
+    });
+
+    describe('with a live query state model', () => {
```
Comment (samisayegh) [inline unitTests/ui/MLFacet/MLFacetTest.ts]: Is this `describe` block needed considering `initializeComponent` always uses a live query state model?

File: unitTests/ui/MLFacet/MLFacetTest.ts [hunk 2]
```diff
@@ -0,0 +1,184 @@
+import * as Mock from '../../MockEnvironment';
+import { MLFacet } from '../../../src/ui/MLFacet/MLFacet';
+import { IMLFacetOptions } from '../../../src/ui/MLFacet/MLFacetOptions';
+import { IMLFacetValue } from '../../../src/ui/MLFacet/MLFacetValues/MLFacetValue';
+import { MLFacetTestUtils } from './MLFacetTestUtils';
+import { $$ } from '../../../src/Core';
+
+export function MLFacetTest() {
```
Comment (samisayegh) [inline unitTests/ui/MLFacet/MLFacetTest.ts]: Good job on these tests 👍

File: unitTests/ui/MLFacet/MLFacetTest.ts (hunk 1 above)
Comment (ThibodeauJF) [reply unitTests/ui/MLFacet/MLFacetTest.ts]: right, I should remove it now that I always use the advanced mock with the qsm

File: unitTests/ui/MLFacet/MLFacetTest.ts (hunk 2 above)
Comment (ThibodeauJF) [reply unitTests/ui/MLFacet/MLFacetTest.ts]: Thanks! It's only a start, much more features to come

**Pull request merged**